apps/backend/src/database/database.module.ts

```diff
@@ -25,12 +23,26 @@ const getEnvironmentSpecificConfig = (env?: string) => {
   switch (env) {
     case 'production':
       return {
+        synchronize: false,
+        migrationsRun: false,
         entities: [join(__dirname, '../**/*.entity.js')],
         migrations: [join(__dirname, '../migration/*.js')],
         logging: ['migration'] as LoggerOptions
       };
+    case 'test':
+      return {
+        synchronize: true,
```

Comment:
sync: true - reasoning?

Reply:
Intention is to recreate a temporary testing db in the container on each run in the pipeline so I thought we'd need to set sync true in order to not run the migrations. Please let me know if there is a better way to configure this?